hooks/create_move/movement.c

```diff
@@ -0,0 +1,54 @@
+#include "../../config/config.h"
+#include "../../utils/utils.h"
+#include "../../source_sdk/entity/entity.h"
+#include "../../source_sdk/math/vec3.h"
+#include "../../source_sdk/user_cmd.h"
+#include "../paint_traverse/paint_traverse.h"
+#include "create_move.h"
+
+void bhop(void *localplayer, struct user_cmd *user_cmd){
+
+    /* scouts double jump only works sometimes, very inconsistent
```

Comment:
You can leave it enabled for scout, i just prefer not having it than having it only work 50% of the time

Reply:
I think autostrafe should be disabled for scout as well then